docs/variants/asus_kgpe_d16/recovery.md

```diff
@@ -0,0 +1,22 @@
+# Recovery
+
+---
+
+## **Prequisitions**
+
+1. To proceed with recovery `.rom` file with backup vendor firmware will be
```

Comment:
Maybe we should change the formatting of this point?

Reply:
Changed to a blank text